DataManagementSystem/Client/DataManager.py

```diff
@@ -463,8 +463,9 @@ def putAndRegister( self, lfn, fileName, diracSE, guid = None, path = None, chec
         errStr = "The supplied LFN already exists in the File Catalog."
         log.debug( errStr, lfn )
       else:
-        errStr = "This file GUID already exists for another file. " \
-            "Please remove it and try again."
+        # If the returned LFN is different, this is the name of a file
+        # with the same GUID
+        errStr = "This file GUID already exists for another file"
         log.debug( errStr, res['Value']['Successful'][lfn] )
       return S_ERROR( "%s %s" % ( errStr, res['Value']['Successful'][lfn] ) )
```

Comment:
I'd split this in:

```
log.error( errStr, "Existing file with same GUID: %s" % res['Value']['Successful'][lfn] )
return S_ERROR( errStr )
```

Because it's more "DError" style